config/initializers/devise.rb

```diff
@@ -232,7 +232,7 @@
   # Turn scoped views on. Before rendering "sessions/new", it will first check for
   # "users/sessions/new". It's turned off by default because it's slower if you
   # are using only default views.
-  # config.scoped_views = false
+  # config.scoped_views = true
```

Comment:
Does this line need to be uncommented?

Reply:
Good catch!  Can actually revert this to false and leave it commented out.  This is config for using per model file paths for devise.  They're all pulling from the same views so I can revert this.